Channels/Integration/01d_Amazon.md

```diff
@@ -0,0 +1,220 @@
+# Manage the Amazon connection
+
+To establish a connection to an Amazon shop, there are several particularities to consider. Therefore, the creation and the configuration of the Amazon connection are described in detail below.
+
+
+## Configuration of the Amazon Seller Account
+
+Before a connection between Amazon and the *Actindo Core1 Platform* can be established, the following adjustments must be made in the Amazon Seller Central for the connection to work correctly:
+
+- All columns of the order reports must be activated  
+- A standard ship-from address must be set
+
+Follow the instructions below to configure your Amazon Seller Account.
+
+[commment]: <> (Andere pre-settings? RS Johannes. S. https://actindo.atlassian.net/wiki/spaces/DEVELOPMEN/pages/21561363/Amazon)
+
+#### Prerequisites  
+
+An Amazon Seller Account has been created.
+
+#### Procedure
+
+1. In a browser window, go to the *Amazon Seller Central* platform and log in to your account.
+
+2. In your Seller profile, go to *Orders > Order reports > Add or remove order report columns*.
+
+  ![Order reports](../../Assets/Screenshots/Channels/Settings/Connections/Amazon/.png "[Order reports]")
+
+3. Add all columns to the order reports. Otherwise, the shipping costs may not be imported.
+
+4. Click the [Save changes] button.
+
+5. Now go to *Settings > Shipping Settings > General Shipping Settings*.
+
+  ![Shipping settings](../../Assets/Screenshots/Channels/Settings/Connections/Amazon/.png "[Shipping settings]")
+
+6. Click the [Edit] icon next to the *Ship from location*.
+
+7. Enter the default ship-from address.
+
+8. Click the [Save] button.
+
+
+[comment]: <> (RS Johannes - Procedure und Screenshots? S. auch https://internal-jira.actindo.com/browse/ACDP-150)
```

Comment:
Ich nehme an, die Info hast du aus dem Confluence Eintrag.
Würde es aber nicht ausreichen, wenn ich die beiden Punkte  
- All columns of the order reports must be activated  
- A standard ship-from address must be set
mit Link auf die Amazon Doku als prerequisites für create an amazon connection angebe?
Brauche ich da wirklich ne eigene Procedure? (finde ich immer schwierig, wenn alles Beschriebene im Fremdsystem stattfindet und es eine Doku gibt...